packages/manager/modules/vrack/src/dashboard/vrack.constant.js

```diff
@@ -26,13 +26,44 @@ export const VRACK_URLS = {
   },
 };
 
+/**
+ * OVH services names as constants
+ * @type {string}
+ */
+export const IP_SERVICE = 'ip';
+export const IPLB_SERVICE = 'ipLoadbalancing';
+export const DEDICATED_SERVICE = 'dedicatedServer';
+export const DC_SERVICE = 'dedicatedConnect';
+export const DSI_SERVICE = 'dedicatedServerInterface';
+export const DCLOUDDC_SERVICE = 'dedicatedCloudDatacenter';
+export const DCLOUD_SERVICE = 'dedicatedCloud';
+export const OVHCC_SERVICE = 'ovhCloudConnect';
+export const CP_SERVICE = 'cloudProject';
+export const LVRACK_SERVICE = 'legacyVrack';
+export const MBM_SERVICE = 'managedBareMetal';
+export const MBMDC_SERVICE = 'managedBareMetalDatacenter';
+export const SERVICES_NAMES = {
+  CP_SERVICE,
+  DCLOUDDC_SERVICE,
+  DCLOUD_SERVICE,
+  DC_SERVICE,
+  DEDICATED_SERVICE,
+  DSI_SERVICE,
+  IPLB_SERVICE,
+  IP_SERVICE,
+  LVRACK_SERVICE,
+  MBMDC_SERVICE,
+  MBM_SERVICE,
+  OVHCC_SERVICE,
+};
```

Comment:
Are you sure you need the both ? It's best to keep the object i think

Reply:
i made the object using the constants, and changed the controller to use the object who gather the constants instead of importing each constant separately